community/boilerplates/serverless-triggers/aws-lambda/go/echo/echo.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+type TableStruct struct {
+	Name string `json:"name"`
+}
+
+type EventStruct struct {
+	Operation string `json:"op"`
+	Data      map[string]map[string]string
```

Comment:
```json
"data": {
  "old": {
    "id": 1,
    "note": "some note string"
  },
  "new": {
    "id": 1,
    "note": "some other note string"
  }
}
```

This is a sample json under `event` key. This does not match with `map[string]map[string]string`

To keep it generic, you can make it like this:

```go
type EventStruct struct {
  Operation string    `json:"op"`
  Data      EventData `json:"data"`
}

type EventData struct {
  Old map[string]interface{} `json:"old"`
  New map[string]interface{} `json:"new"`
}
```